modules/nixos/common/default.nix

```diff
@@ -40,6 +40,15 @@
 
   zramSwap.enable = true;
 
+  # https://github.com/NixOS/nixpkgs/pull/268121
+  # https://wiki.archlinux.org/title/Zram#Optimizing_swap_on_zram
+  boot.kernel.sysctl = {
+    "vm.page-cluster" = 0;
```

Comment:
Are we using zstd in our case? The linked pull request does set this value conditionally.

Reply:
Yes, zstd is the default.